ocaml/runtime4/alloc.c

```diff
@@ -67,7 +67,7 @@ CAMLexport value caml_alloc (mlsize_t wosize, tag_t tag) {
   return caml_alloc_with_reserved (wosize, tag, 0);
 }
 
-#if NATIVE_CODE
+#if defined(NATIVE_CODE) && NATIVE_CODE
```

Comment:
(There are occurrences of `#ifdef NATIVE_CODE` under `ocaml/runtime`.)

Reply:
(I misread this comment.)  Just using `#ifdef`.